packages/studio-base/src/Workspace.tsx

```diff
@@ -599,24 +599,24 @@ function WorkspaceContent(props: WorkspaceProps): JSX.Element {
 
   const leftSidebarItems = useMemo(() => {
     const items = new Map<LeftSidebarItemKey, NewSidebarItem>([
+      ["panel-settings", { title: "Panel settings", component: PanelSettingsSidebar }],
```

Comment:
```suggestion
      ["panel-settings", { title: "Panel", component: PanelSettingsSidebar }],
```

@2metres wdyt? Seems cleaner imo, and still clear what it does